docs/modules/storage/pages/configuring-persistence.adoc

```diff
@@ -1261,6 +1261,55 @@ config.addMapConfig(mapConfig);
 ====
 |===
 
+[[sql]]
+== SQL Metadata Options
+
+Use the following configuration to enable the cluster-wide persistence of SQL metadata to disk. With persistence enabled, SQL mappings, data connections, and views are retained after the cluster restarts.
+
```

Comment:
Something I forgot to mention, but we probably want to add a note here: 

```
NOTE: A known issue is that for clusters larger than 6 members, it might take some time for the SQL Catalog to become available on all nodes after a restart, which may impact Hot Restart of jobs that use catalog entities (e.g. jobs that use Data Connections).
```

Its a known bug that unfortunately could not be fixed in 5.3.0.

Reply:
Added note.